vite-project-kabinet/src/Content/PaginationNav.jsx

```diff
@@ -0,0 +1,34 @@
+import arrowRight from "./assets/arrow_right.svg"
+import arrowLeft from "./assets/arrow_left.svg"
+
+export const PaginationNav = ({currentPage, totalPages, handlePageChange}) => {
+
+    return (
+        <div className="pages_pc">
```

Comment:
Někde označuješ široké obrazovky jako _desktop_ jinde jako _pc_. Přesnější je _desktop_, ale ani to není přesné, protože větší telefon nebo tablet na šířku ti spadnou do stejné kategorie. Lepší je o tom nepřemýšlet jako o zařízení (mobil, desktop) ale jako o šířce viewportu (small, medium, large, extra large...). Většina frameworků funguje právě takhle, protože to je to co Tě zajímá. Jak široká (obecně velká) obrazovka je, ne jestli je to počítač, tablet, televize...

Ale hlavně buď konzistentní v tom, jak tomu říkáš :-)